test/Sentry.Testing/DsnSamples.cs

```diff
@@ -16,3 +16,20 @@ public static class DsnSamples
     /// </summary>
     public const string InvalidDsn = "https://d4d82fc1c2c4032a83f3a29aa3a3aff@fake-sentry.io:65535/";
 }
+
+public class ValidDsnNoSecretOptions : SentryOptions
+{
+    public ValidDsnNoSecretOptions()
+    {
+        Dsn = DsnSamples.ValidDsnWithoutSecret;
+    }
+}
+
+public class ValidDsnOptions : SentryOptions
+{
+    public static ValidDsnOptions Instance = new ValidDsnOptions();
+    public ValidDsnOptions()
+    {
+        Dsn = DsnSamples.ValidDsnWithSecret;
```

Comment:
Secret has been deprecated so many years ago that we can just delete it at this point. The SDK requires the `/envelope` endpoint so doesn't support old Sentry's that would require a secret in the DSN.